aisynphys/ui/notebook.py

```diff
@@ -173,11 +173,16 @@ def show_connectivity_matrix(ax, results, pre_cell_classes, post_cell_classes, c
     for i,pre_class in enumerate(pre_cell_classes):
         for j,post_class in enumerate(post_cell_classes):
             result = results[pre_class, post_class]
-            cp, cp_lower_ci, cp_upper_ci = result[ctype]
-            if ctype == 'gap_probability':
-                found = result['n_gaps']
-            else:
+            if ctype == 'chemical':
                 found = result['n_connected']
+                if distance_adjusted:
+                    cp, cp_lower_ci, cp_upper_ci = result['adjusted_connectivity']
+                else:
+                    cp, cp_lower_ci, cp_upper_ci = result['connection_probability']
+            if ctype == 'electrical':
+                found = result['n_gaps']
```

Comment:
check for `distance_adjusted` here and either return adjusted values or raise an exception saying this isn't implemented yet.